modules/solid_mechanics/src/userobjects/CrackFrontDefinition.C

```diff
@@ -22,6 +22,7 @@ void addCrackFrontDefinitionParams(InputParameters& params)
 {
   MooseEnum direction_method("CrackDirectionVector CrackMouth CurvedCrackFront");
   MooseEnum end_direction_method("NoSpecialTreatment CrackDirectionVector", "NoSpecialTreatment");
+  params.addParam<std::vector<RealVectorValue> >("crack_front_points","Set of points to define crack front");
```

Comment:
I'm saying the you can change the type from `RealVectorValue` to `Point` in both the .h file and the `validParams` function and it'll work the exact same way as it does now (i.e. reads in three points at a time and stores them as a vector of Points). You don't gain extra functionality but the types are correct (i.e. You are storing Points as Points instead of RealVectorValues). I'll leave the decision to change it up to you, I just wanted to make you aware that we support vectors of Points too.